src/index.css

```diff
@@ -201,9 +201,8 @@ footer {
   .navbar__header {
     padding: 0 2rem;
   }
-  
+
   .container {
     padding: 2em 1em !important;
   }
-
 }
```

Comment:
Well done working on your project, you've implemented most of the requirements. you still have 2 small isuues to fix:
- [ ] The live version is not working correctly
- [ ] the two color shades:. The Behance design uses two alternating shades of a color for the items on the home page, you are now using 2 color shades but try to alternate them as per Behance design as requested before.


<img width="442" alt="Screenshot 2023-09-09 at 3 57 21 PM" src="https://github.com/chudisoft/metrics-webapp/assets/53347966/442a63ee-82f4-4d7f-aa30-f1a145ad07d7">